ui/src/main/resources/CKEditor/Translations.xml

```diff
@@ -58,7 +58,7 @@ ckeditor.admin.reset.confirmation=Are you sure you want to restore the default c
 ckeditor.plugin.link.options=Options
 ckeditor.plugin.link.anchor=Anchor
 ckeditor.plugin.link.queryString=Query String
-ckeditor.plugin.link.defaultLabel=type the link label
+ckeditor.plugin.link.defaultLabel=Type the link label
```

Comment:
Marius refers that if you insert a label using the modal and leave the default value it will get inserted (in most cases) inside a phrase, not at the beginning (to justify the capitalization), see 
<img width="1053" alt="link" src="https://cloud.githubusercontent.com/assets/629552/25655370/4ef6a752-2ffd-11e7-88f8-a289b82bcc54.png">

Now the question is: how many users edit the link label inside the modal vs. inside the editor textarea?